prisma/schema.prisma

```diff
@@ -8,6 +8,18 @@ generator client {
 }
 
 model User {
-  id    Int    @id @default(autoincrement())
-  email String
+  id     Int      @id @default(autoincrement())
```

Comment:
Is it safe to users have a predictable numerical id associated with them? I think GUIDs might be the fashion now days 🤔?

Reply:
Good question. My understanding of the pros of using a GUID would be:

- Unique across multiple DBs/servers
- If a full user object is used client side, GUIDs give bad actors less info

Would you suggest using a GUID as the PK here?

Reply:
If you're up for the challenge, yes.

Reply:
Pretty easy change with prisma